_publications/jors/paper.tex

```diff
@@ -0,0 +1,457 @@
+communities. The first deployment of the PFHub framework supports
+phase-field practitioners and code developers participating in an
+effort to improve quality assurance for phase-field codes.
+
+\section*{Keywords}
+
+phase-field; materials-science; jekyll-website; reproducible-science
+
+\section*{Introduction}
+
+Generally, small scientific communities do not have the resources to
+build and host dedicated web infrastructure to support their varied
+content and data requirements. In particular, hosting and supporting a
+complex content management system (CMS) including web servers, web
+frameworks and databases requires a great deal of configuration and
+long term support and funding. Furthermore, a turnkey CMS solution may
+not meet requirements for most scientific communities that often use
+arcane data formats and require custom data displays along with
+client-side automation. The PFHub effort, instead of focusing on the
+CMS tool, focuses on customizing and delivering the client-side
+requirements whilst delegating back-end functionality to external
+services that provide dependable APIs~\cite{cmsfree}.
+
+The phase field method (PFM) describes material interfaces at the
+mesoscopic scale between atomic scale models and macroscale
+models~\cite{MOELANS2008268}. The PFM is well established and there
+are an assortment of code frameworks (\emph{e.g.}, FiPy~\cite{fipy},
+MMSP~\cite{mmsp}, MOOSE~\cite{moose}, PRISMS-PF~\cite{prisms-pf})
+available for solving the wide variety of phenomena associated with
+phase field (e.g. dendritic growth, spinodal decomposition, grain
+growth)~\cite{tonks}. However, it is difficult for novice as well as
+seasoned phase field practitioners to asses the capability of codes
+for different phenomena without extensive prototyping and
+groundwork. PFHub aims to provide a low barrier for comparing code
+output data using a standard set of metrics.
+
+PFHub is a community effort spearheaded by the Center for Hierarchical
+Materials Design at Northwestern University and the National Institute
+of Standards and Technology in support of phase-field code
+development. The current PFHub deployment~\cite{pfhub} focuses on
+improving cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark
+problems~\cite{bm1, bm2} and a workflow for uploading and comparing
+benchmark results from different phase-field codes.
+
+Community based scientific efforts often require web services to share
+and display data in unique ways between groups and institutions. These
+services are difficult to implement due to the groundwork required to
+investigate and prototype the many data-sharing and CMS tools
+available. The PFHub framework provides a template for other
+scientific projects beyond the phase field community. The method
+outlined in this paper of using static infrastructure coupled with
+small independent 3rd party web services providing a flexible approach
```

Comment:
I'd suggest "provides" instead of "providing" here

Reply:
Fixed